gufe/protocols/protocoldag.py

```diff
@@ -359,6 +359,7 @@ def execute_DAG(protocoldag: ProtocolDAG, *,
                 keep_shared: bool = False,
                 keep_scratch: bool = False,
                 raise_error: bool = True,
+                n_retries: int = 0,
```

Comment:
Are we changing the default in quickrun?

Reply:
Yes, different repo but 2 retries would make sense?

Reply:
Yeah my question was more "here or there".